### DISCUSSION TITLE
Invalid copilot token: missing token: 403  Extension activation failed: "Contact Support. You are currently logged in

### DISCUSSION THREAD

**Discussion by Micdiane** [community/community#114348] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

How can I slove this problem, I have the right in using Copilot , and try other methods, Invalid copilot token: missing token: 403  Extension activation failed: "Contact Support. You are currently logged in

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
